webapp/src/app.tsx

```diff
@@ -3730,6 +3743,14 @@ export class ProjectView
         return null;
     }
 
+    private saveTutorialTemplateAsync() {
+        return pkg.mainEditorPkg().buildAssetsAsync()
+            .then(() => pkg.mainPkg.saveToJsonAsync())
+            .then(project => {
+                pxt.BrowserUtils.browserDownloadText(project.source, pkg.genFileName(".txt.mkcd"))
```

Comment:
```suggestion
                pxt.BrowserUtils.browserDownloadText(project.source, pkg.genFileName(".assets.mkcd"))
```

Fine either way, but maybe a bit more descriptive than .txt? I also personally get weirded out  a bit when I see `filename.{common file extension}.{something else}` as people occasionally try and use that for viruses (`famous-book.pdf.exe`).